### PR TITLE
Activity: tweak threats styling

### PR DESCRIPTION
**Pull request by keoshi**
This set of changes tweak the threats styling and change some language to match the current patterns.

Notably:

- Reducing the icon size.
- Tweak overall padding of card.
- Add red alert border.
- Make font weights consistent across the board.
- Reorder description to the top of the expanded section.

### Before

screen shot 2018-05-17 at 11 54 53

### After

screen shot 2018-05-17 at 13 07 17

### CODE REVIEW

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 1]
```diff
@@ -101,10 +101,10 @@ const headerSubtitle = ( translate, threat ) => {
 			} );
 
 		case 'plugin':
-			return translate( 'Vulnerability found in Plugin' );
+			return translate( 'Vulnerability Found in Plugin' );
 
 		case 'theme':
-			return translate( 'Vulnerability found in Theme' );
+			return translate( 'Vulnerability Found in Theme' );
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 2]
```diff
@@ -101,10 +101,10 @@ const headerSubtitle = ( translate, threat ) => {
 			} );
 
 		case 'plugin':
-			return translate( 'Vulnerability found in Plugin' );
+			return translate( 'Vulnerability Found in Plugin' );
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 3]
```diff
@@ -89,10 +89,10 @@ const headerTitle = ( translate, threat ) => {
 const headerSubtitle = ( translate, threat ) => {
 	switch ( detailType( threat ) ) {
 		case 'core':
-			return translate( 'Vulnerability found in WordPress' );
+			return translate( 'Vulnerability Found in WordPress' );
 
 		case 'file':
-			return translate( 'Threat found ({{signature/}})', {
+			return translate( 'Threat Found ({{signature/}})', {
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 4]
```diff
@@ -89,10 +89,10 @@ const headerTitle = ( translate, threat ) => {
 const headerSubtitle = ( translate, threat ) => {
 	switch ( detailType( threat ) ) {
 		case 'core':
-			return translate( 'Vulnerability found in WordPress' );
+			return translate( 'Vulnerability Found in WordPress' );
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/stats/activity-log/style.scss [hunk 5]
```diff
@@ -79,37 +79,56 @@
...
-	.activity-log__threat-alert-title {
-		color: black;
-	}
+.activity-log__threat-alert-title {
+	font-weight: 500;
+	color: $gray-dark;
 
-	.activity-log__threat-alert-slug {
-		font-weight: bold;
+	code {
```
[the remark this answers is not in the document]
Comment (keoshi) [reply client/my-sites/stats/activity-log/style.scss]: Oops, yeah — I should probably make that a selector. Thanks for catching that!

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 6]
```diff
@@ -101,10 +101,10 @@ const headerSubtitle = ( translate, threat ) => {
 			} );
 
 		case 'plugin':
-			return translate( 'Vulnerability found in Plugin' );
+			return translate( 'Vulnerability found in plugin' );
 
 		case 'theme':
-			return translate( 'Vulnerability found in Theme' );
+			return translate( 'Vulnerability found in theme' );
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

File: client/my-sites/stats/activity-log/threat-alert.jsx [hunk 7]
```diff
@@ -101,10 +101,10 @@ const headerSubtitle = ( translate, threat ) => {
 			} );
 
 		case 'plugin':
-			return translate( 'Vulnerability found in Plugin' );
+			return translate( 'Vulnerability found in plugin' );
```
Comment (a8ci18n) [inline client/my-sites/stats/activity-log/threat-alert.jsx]: :ok: This change is safe, we'll keep the existing 17 translations.

**Pull request merged**